src/SerializableClosure.php

```diff
@@ -107,8 +128,22 @@ public function bindTo($newthis, $newscope = 'static')
     public function serialize()
     {
         try {
-            $this->data = $this->data ?: $this->serializer->getData($this->closure, true);
-            return serialize($this->data);
+            $this->data = $data = $this->data ?: $this->serializer->getData($this->closure, true);
+
+            if ($this->compressClosure) {
+                $file = tempnam('', '');
+                $code = $data['code'];
+
+                file_put_contents($file, '<?php ' . $code);
+
+                exec("php -w {$file}", $output);
```

Comment:
You really wanna gzip rather than doing this sort of simplified whitespace compression :X